pkg/glusterfs/Dockerfile

```diff
@@ -10,55 +10,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#-- Create build environment
 
-#-- Build phase
-FROM openshift/origin-release:golang-1.10 AS build
+FROM docker.io/openshift/origin-release:golang-1.10
```

Comment:
FROM ... AS is supported. The following works for me:

```
$ cat Dockerfile 
FROM centos:7.5.1804 as base
RUN echo "hello" > /h

FROM centos:7.5.1804 as final
COPY --from=base /h /h
RUN cat /h
```
with
```
$ buildah bud .
STEP 1: FROM centos:7.5.1804
Getting image source signatures
Copying blob sha256:7dc0dca2b1516961d6b3200564049db0a6e0410b370bb2189e2efae0d368616f
 71.23 MiB / 71.23 MiB [===================================================] 19s
Copying config sha256:76d6bc25b8a5685072a1a99d9ac7c2e52dc3070081c872034a1889ca2d4bcf8c
 2.15 KiB / 2.15 KiB [======================================================] 0s
Writing manifest to image destination
Storing signatures
STEP 2: RUN echo "hello" > /h
STEP 3: FROM centos:7.5.1804
STEP 4: COPY --from=base /h /h
STEP 5: RUN cat /h
hello
STEP 6: COMMIT containers-storage:[vfs@/home/jstrunk/.local/share/containers/storage+/run/user/101587/run]@4a5769081c038b6ed9e8fe400078d068e773c6ed048ff485e648c2f1985113c2
Getting image source signatures
Skipping fetch of repeat blob sha256:bcc97fbfc9e1a709f0eb78c1da59caeb65f43dc32cd5deeb12b8c1784e5b8237
Copying blob sha256:34e18c749bcf5457f2a2d4d36a3a0fce5e04f723c81be2dc38f37835256dbf27
 169 B / 169 B [============================================================] 0s
Copying config sha256:fcf270bec64248fd75cbe464ce99bfc0229c56ba2fdb7894a100532a97fd07cd
 1.18 KiB / 1.18 KiB [======================================================] 0s
Writing manifest to image destination
Storing signatures
--> 4a5769081c038b6ed9e8fe400078d068e773c6ed048ff485e648c2f1985113c2
```

Reply:
Fair enough. I didn't see the point of keeping it if we're not making meaningful use of it, but I'll add it back in.